Makefile.am

```diff
@@ -45,3 +45,12 @@ clr_debug_daemon_LDADD = ${curl_LIBS} libnica.la ${LIBSYSTEMD_LIBS}
 systemdsystemunit_DATA = clr_debug_fuse.service clr_debug_daemon.service clr_debug_daemon.socket
 
 tmpfiles_DATA = debuginfo.conf
+
+# Functional Testing
+noinst_PROGRAMS = testing_fuse testing_daemon
```

Comment:
Can you use `check_PROGRAMS` instead, so that the binaries can be run with `make check`?

Reply:
the binaries cannot being run in make check because then they are going to fail in the github CI, this tests needs fuse kernel module installed and root grants thing that We cannot do to the github container if we don't have a custom container.

Reply:
Oh, you're right, I didn't see that `make distcheck` is currently run in the Github Actions. Disregard this suggestion for now :-)